### ISSUE TITLE
Spend unconfirmed with sendcoins api, for CPFP,

### ISSUE THREAD

**Issue by sako0938** [lightningnetwork/lnd#4158]
### Background

I am trying to create a child-pays-for-parent transaction, using the sendcoins api function, and spend an unconfirmed output, so I can bump up the fee. You are able to spend unconfirmed outputs when opening a channel, I think you should also be able to do the same with sendcoins on-chain function.

Ive checked all of the different APIs (CLI, REST, and gRPC) and the feature isnt available in any of them. I mostly desire this feature for usage in the REST API.

### Your environment

LND Master REST API

### Expected behaviour

```
def sendCoins(addr,amt,toself=True):
	url = '/v1/transactions'
	if toself:
		addr = getNewAddress()
	# either target_conf or sat_per_byte used at one time
	data = { 
		# 'target_conf': 20,
		'sat_per_byte': 5, 
		'send_all': False, 
		'addr': f'{addr}', 
		'amount': f'{amt}',
		'spend_unconfirmed': True
	}
	lnreq = sendPostRequest(url,data)
	return lnreq```

### Actual behaviour

{'error': 'insufficient funds available to construct transaction', 'message': 'insufficient funds available to construct transaction', 'code': 2, 'details': []}

**Comment by wpaulino** [comment]
Check out [`BumpFee`](https://github.com/lightningnetwork/lnd/blob/173f7d88e0ef7a2bfe1d8c44a67fd8d1ad52a357/lnrpc/walletrpc/walletkit.proto#L301) available in the `WalletKit` subserver. You'll need to compile `lnd` with `make install tags=walletrpc` to enable it. It's only available with gRPC at the moment, but we are working on supporting REST as well.

**Comment by Roasbeef** [comment]
Closing as we have an API available for this. By the time 0.10 is released, more of these APIs will be better documented on our main API site.

**Comment by sako0938** [comment]
Does that mean that the REST API for this will be available in 0.10.0? If not, the issue I think is still relevant.
